src/main/java/decodes/tsdb/compedit/AlgorithmsListPanel.java

```diff
@@ -58,45 +63,68 @@ JPanel getFieldPanel() {
 		fillLabels();
 		this.add(getJContentPane(), java.awt.BorderLayout.CENTER);
 	}
-	
+
 	private void fillLabels()
 	{
 		openErr = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.OpenError");
 		newInputText = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.NewInputText");
 		newError = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.NewError");
 		cpyErr1 = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.CopyError1");
-		cpyErr2 = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.CopyError2");	
+		cpyErr2 = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.CopyError2");
 		cpyInput = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.CopyInput");
 		deleteErr1 = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.DeleteError1");
 		deleteErr2 = CAPEdit.instance().compeditDescriptions.getString("AlgorithmsListPanel.DeleteError2");
 	}
 
 	/**
 	 * This method initializes jContentPane
-	 * 
+	 *
 	 * @return javax.swing.JPanel
 	 */
-	protected JPanel getJContentPane() 
+	protected JPanel getJContentPane()
 	{
-		if (jContentPane == null) 
+		if (jContentPane == null)
 		{
 			jContentPane = new JPanel();
 			jContentPane.setLayout(new BorderLayout());
-			jContentPane.add(getButtonPanel(), java.awt.BorderLayout.SOUTH);
+			JPanel buttonPanel = getButtonPanel();
+			JButton checkForNew = new JButton(ResourceBundle.getBundle("ilex/resources/gui").getString("EditorMenuSet.checkForNew"));
```

Comment:
Suggestion on the naming, method checkForNew() and label 'Check for new'

How about: 'Refresh' ;   I admit I'm confused on what is being synced here.

Reply:
It's definitely not a "refresh", that's used in other GUIs to reload from the database. It's specifically checking for new possible algorithms from various sources.